spyder/plugins/run/container.py

```diff
@@ -88,6 +90,12 @@ def setup(self):
             context=Qt.ApplicationShortcut
         )
 
+        self.create_action(
+            RunActions.GlobalConfigurations,
+            _("Global configurations"),
```

Comment:
Maybe it could be worthy to add a menubar shortcut to this new global action too? So:
```suggestion
            _("&Global configurations"),
```